docs/tutorials/mint-farm-yusdbtc.md

```diff
@@ -0,0 +1,151 @@
+
+A one-time deposit address is generated for you to send your BTC to. Send the exact amount of BTC you are wrapping to that address, in a single transaction.
+
+![deposit](https://github.com/henrystats/docs/blob/master/static/img/Screenshot%20(142).png)
+
+* Your transaction requires 6confirmations(can take up to 1hour) before renBTC is sent to your wallet 
+* You will need to accept & sign the transaction that sends renBTC to your wallet.
+
+
+
+## Step 2: Minting yUSD with renBTC
+
+Minting yUSD tokens is easy with the [EMP Tools](http://tools.umaproject.org/),
+UMA's UI for ExpiringMultiParty contracts (EMP) which is what the yUSD token is
+minted from.
+
+Navigate to http://tools.umaproject.org/ and click the "Connect" button at the
+top right corner to connect your MetaMask wallet to the dapp.
+
+![connect](/docs/tutorials/emp_connect.png)
+
+Then, select the `yUSDBTC` contract from the dropdown immediately underneath the
+header (above the tabs).
+
+![select EMP](/docs/tutorials/emp_select.png)
+
+Select the "Manage Position" tab.
+
+![manage position tab](/docs/tutorials/emp_manage-position.png)
+
+Scroll down to the "Actions" dropdown and ensure that "Create" is selected.
+
+![position actions dropdown](/docs/tutorials/emp_actions.png)
+
+At this point, make sure you read the text and confirm that:
+
+1. You will be minting at a collateralization ratio above the global
+   collateralization ratio (GCR);
+2. You will be minting the minimum required number of tokens (100 for `yUSD`),
+   and;
+3. You will keep your position collateralized above the minimum required
+   collateralization ratio (CR) or else risk getting liquidated (the required CR
+   is 125% for `yUSD`).
+
+If you fully understand the above, and have decided on an appropriate amount of
+(1) collateral to supply and (2) tokens to mint, check to make sure you have
+enough collateral for minting.
+
+![token minting form](/docs/tutorials/emp_form.png)
+
+In the first field, fill in the amount of collateral you want to supply.
+
+In the second field, fill in the number of tokens you want to mint.
+
+If these values are appropriate, you can then click the "Create" button which
+will trigger a transaction with MetaMask. Once that is confirmed and the
+transaction is mined, you shall have your `yUSD` tokens!
+
+## Step 3: Farming yUSD on Balancer
+
+:::danger
+
+Supplying _only_ `USDC` to the pool is very dangeorus and is not recommended,
+because you are essentially buying `yUSD` on the open market. You should avoid
+this because `yUSD` will be worth
+$1 by the time of expiry and you will lose
+money if you are buying `yUSD` above $1.
+
+:::
+
+By supplying liquidity to the `yUSD/USDC` Balancer pool, you have the
+opportunity to earn `BAL`, `REN` and `UMA` tokens. The `UMA` token distribution
+is governed by the experimental UMA liquidity mining program.
+
+[INSERT LINK TO RENBTC LIQUIDITY MINING PROGRAM BLOG POST]
```

Comment:
I think this link is breaking CI. This is the output from Vercel:
![image](https://user-images.githubusercontent.com/12886084/92224627-5cf69d00-eea2-11ea-8b81-89307c9bedce.png)